code/datums/wounds/bones.dm

```diff
@@ -0,0 +1,209 @@
+
+
+
+/*
+	Bones
+*/
+// TODO: well, a lot really, but i'd kill to get overlays and a bonebreaking effect like Blitz: The League, similar to electric shock skeletons
+
+
+/datum/wound/brute/bone
+	sound_effect = 'sound/effects/crack1.ogg'
+	/// If a bone wound is applied to a leg, we store the limp component here so we can delete it when removed
+	var/datum/component/limp/current_limp
+	wound_type = WOUND_TYPE_BONE
+	/// How effective various item types are at splinting these injuries. Checks from left to right, subpaths are accepted, so put specific subtypes before general ones
+	var/list/splint_items = list(/obj/item/stack/medical/gauze = 0.25, /obj/item/stack/sticky_tape/surgical = 0.4, /obj/item/stack/sticky_tape/super = 0.6, /obj/item/stack/sticky_tape = 0.75)
```

Comment:
Put the redefined vars on top, and the new variables under, for clarity.

Reply:
Done, I'll make sure to observe this in the future